packages/runtime/src/__tests__/provider.test.tsx

```diff
@@ -0,0 +1,126 @@
+const mockFunction = jest.fn();
+
+jest.mock('../is-node', () => ({
+  isNodeEnvironment: mockFunction,
+}));
+
+const OLD_ENV = process.env;
+
+describe('Provider index.tsx', () => {
+  beforeEach(() => {
+    jest.resetModules(); // most important - it clears the cache
+    process.env = { ...OLD_ENV }; // make a copy
+    process.env.NODE_ENV = 'development';
+  });
+  afterAll(() => {
+    process.env = OLD_ENV; // restore old env
+  });
+  it('returns the browser provider when isNodeEnvironment returns false', () => {
+    mockFunction.mockImplementation(() => false);
+    const loadComp = (path, name = 'default') => {
+      let comp = undefined;
+      try {
+        comp = require(path)[name];
+      } catch (_) {}
+      return comp;
+    };
+    const BrowserProvider = loadComp('../provider/provider-browser');
```

Comment:
can we use require here instead of the load comp wrapper?

Reply:
Done